src/components/header-component/header.jsx

```diff
@@ -1,26 +1,25 @@
 import React from "react";
 import { Link } from "react-router-dom";
-import {ReactComponent as Logo} from '../../assets/crown.svg';
+import { ReactComponent as Logo } from '../../assets/crown.svg';
 import './header.styles.css';
 
-const Header = () => (
+const Header = (props) => (
     <div className='header row'>
-        <div className='col-2 mt-4'>  
+        <div className='col-2 mt-4'>
             <a className='logo-container' to="/">
-            <Logo className='logo'/>
+                <Logo className='logo' />
             </a>
         </div>
-        <div className='col-2'>  
-        <h2 className='title'>Movie Shop</h2>
+        <div className='col-2'>
+            <h2 className='title'>Movie Shop</h2>
         </div>
-       
-      <div className='col-7'>
-        <div className='options'>
-            <Link className='option' to='/shop'> HOME </Link>
-            <Link className='option' to='/shop'> CONTACT </Link>
+        <div className='col-7'>
+            <div className='options'>
+                <Link className='option' to='/shop'> HOME </Link>
+                <Link className='option' to='/shop'> CONTACT </Link>
```

Comment:
Are you going to create a page for contact? If yes, keep it and proceed! I would love to see how you will do it! Otherwise, you don't need this in your code. Try to keep in your UI things that actually work!